try:
    print(3/0)
except Exception as e:
    print('popo',e)
